plugins/eventMacro/eventMacro/Core.pm

```diff
@@ -195,22 +195,29 @@ sub create_automacro_list {
 		}
 		
 		PARAMETER: foreach my $parameter (@{$value->{'parameters'}}) {
-		
 			###Check Duplicate Parameter
 			if (exists $currentParameters{$parameter->{'key'}}) {
 				warning "[eventMacro] Ignoring automacro '$name' (parameter ".$parameter->{'key'}." duplicate)\n";
 				next AUTOMACRO;
 			}
-			###Parameter: call
-			if ($parameter->{'key'} eq "call" && !$self->{Macro_List}->getByName($parameter->{'value'})) {
-				warning "[eventMacro] Ignoring automacro '$name' (call '".$parameter->{'value'}."' is not a valid macro name)\n";
-				next AUTOMACRO;
-			
+			###Parameter: call with or without param
+			if ($parameter->{'key'} eq "call" && $parameter->{'value'} =~ /(\S+)\s+(.*)?/) {
+				if (!$self->{Macro_List}->getByName($1) ) {
+					warning "[eventMacro] Ignoring automacro '$name' (call '".$1."' is not a valid macro name)\n";
+					next AUTOMACRO;
+				} else {
+					if (defined $2) {
```

Comment:
Do we need to do this? It just transforms the sequence of spaces between the name and variables into a single space.

Reply:
now that you said, i've noticed, it's useless, thanks 😄